README.md

```diff
@@ -72,21 +72,41 @@ jobs:
     # the ${{ steps.buildnumber.outputs.build_number }} output.
 ```
 
+## Generating multiple independent build numbers
+
+Sometimes you may have more than one project to build in one repository. For example you may have a client and a server in the same github repository that you would like to generate independent build numbers for. Another example is you have two Dockerfiles in one repo and you'd like to version each of the built images with their own numbers.  
+To do this, use the `prefix` key, like so:
+
+```yaml
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Generate build number
+      id: buildnumber
+      uses: einaregilsson/build-number@v1 
```

Comment:
When this gets merged it would become v2, so the example should say `uses: einaregilsson/build-number@v2`

Reply:
Probably should do that for all examples.

Reply:
All examples changed to v2